simSPI/helpers.py

```diff
@@ -0,0 +1,218 @@
+"""Helper functions for tem.py."""
+import math
+import os
+
+import h5py
+import mdtraj as md
+import mrcfile
+import numpy as np
+from scipy.spatial.distance import pdist
+from scipy.stats import special_ortho_group
+
+
+def mrc2data(mrc_file=None):
+    """mrc2data."""
```

Comment:
We need a legitimate docstring, see our contributing guidelines:
https://github.com/compspi/compspi/blob/master/docs/contributing.rst#writing-docstrings

Can you add well-formatted docstrings to all of these functions? Thanks!